internal/repl/eval.go

```diff
@@ -0,0 +1,37 @@
+package repl
+
+import (
+	"awans.org/aft/internal/db"
+	sh "awans.org/aft/internal/starlark"
+	"fmt"
+	"go.starlark.net/starlark"
+	"strings"
+	"time"
+)
+
+var existing_state starlark.StringDict
```

Comment:
this architecture means there's just a single global repl for all of aft, which I suspect we don't want!

Reply:
Instead, I'd suggest keeping the server stateless, and making it clear to users that each statement is one-shot -- we can't support long-running transactions.